src/source/PeerConnection/Rtcp.c

```diff
@@ -277,44 +311,75 @@ STATUS parseRtcpTwccPacket(PRtcpPacket pRtcpPacket, PTwccManager pTwccManager)
 STATUS onRtcpTwccPacket(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerConnection)
 {
     STATUS retStatus = STATUS_SUCCESS;
-    PTwccManager twcc;
+    PTwccManager pTwccManager = NULL;
     BOOL locked = FALSE;
-    BOOL empty = TRUE;
     UINT64 sn = 0;
```

Comment:
`sn` can be `UINT16` since we're setting it to `prevReportedBaseSeqNum` which is already `UINT16`

Also can we change the name of `sn` to be a bit more descriptive, something like `baseSeqNum` should be fine and that is consistent with what it is called in other functions.

Reply:
Done